src/monika.config.js

```diff
@@ -3,25 +3,26 @@
  * @description: Sysadmins: this is what you will execute using PM2.
  */
 module.exports = {
-    apps: [{
-        name: 'Monika.Gateway',
-        script: './Clara/bot.js',
-        error_file: '../logs/gateway/err.log',
-        out_file: '../logs/gateway/out.log',
-        log_date_format: 'YYYY-MM-DD HH:mm Z',
-        watch: ['./Clara/bot.js'],
-        ignore_watch: ['data', 'logs', 'cmd', 'cache', 'node_modules'],
-        args: ['--color']
-    },
-    {
-        //TODO: Figure out Monika.PoemService's API. It seems ot be HTTP-based
-        name: 'Monika.PoemService',
-        interpreter: '/usr/bin/luajit',
-        script: './ImageGenerator/main.lua',
-        error_file: '../logs/PoemService/err.log',
-        out_file: '../logs/PoemService/out.log',
-        log_date_format: 'YYYY-MM-DD HH:mm Z',
-        ignore_watch: ['data', 'logs', 'cmd', 'cache', 'node_modules'],
-        args: ['--color']
-    }]
+    apps: [
+        {
+            name: 'Monika.Gateway',
+            script: './Clara/bot.js',
+            error_file: '../logs/Gateway/err.log',
+            out_file: '../logs/Gateway/out.log',
+            log_date_format: 'YYYY-MM-DD HH:mm Z',
+            watch: ['./Clara/bot.js'],
+            ignore_watch: ['data', 'logs', 'cmd', 'cache', 'node_modules'],
+            args: ['--color']
+        },
+        {
+            name: 'Monika.PoemService',
+            interpreter: '/user/bin/python3.6',

```

Comment:
isn't the interpreter ``/usr/bin/python36``?

Reply:
I dunno, it's `python3.6` on my system

Reply:
just do ``/usr/bin/python36`` tbh since its more universal